help/2 page

```diff
@@ -0,0 +1 @@
+teach me papa
```

Comment:
```suggestion
teach me papa
```

Reply:
- [ ] how to put 2 pages in 1
- [ ] how to make paths

Reply:
Do you mean two HTML pages?

Reply:
yes

Reply:
both

Reply:
Do you want to make paths in SVG or in CANVAS?  I am guessing is in Canvas since it is for Solid Happiness.